KMarketer.css

```diff
@@ -272,34 +258,396 @@ ul {
         z-index: -1; 
     }
 
-    .main-content {
-        padding: 75px 0px 50px 0px;
+    .video-img {
+        cursor: pointer;
+        transition: opacity 0.5s ease-in-out; /* Устанавливаю плавное изменение прозрачности */
+        opacity: 1;
+        border-radius: 5px;
+        border: 1px;
+    }
+
+    .video-img:hover {
+        opacity: 0.8;
+    }
+}  
+
+/* main-section-cards-reviews */
+
+.cards-reviews {
+    background-color: #252B42; 
+    height: 590px;
+    padding: 80px 0;
+
+    .card-reviews-container {
+        display: flex; /* Отображаю элементы внутри .card-reviews-container в виде flex-контейнера */
```

Comment:
I highly recommend you to create a common "card-container" class which will be used for all blocks that have cards inside. You already have styles for it (display: grid ....)

I agree that here are just 3 elements in one line and flex does the job well. But what if you get one more card? Or 5 more? 
Using grid here will keep you out of trouble.

And of course 1 common class will lead to less code